examples/jsm/utils/BufferGeometryUtils.js

```diff
@@ -106,6 +106,11 @@ function computeMikkTSpaceTangents( geometry, MikkTSpace, negateSign = true ) {
  */
 function mergeGeometries( geometries, useGroups = false ) {
 
+	// Ensure that we have a non-empty array. If not, make the developer aware with an informative error message
+	if (!Array.isArray(geometries) || geometries.length === 0) {
+        throw new Error('mergeGeometries() was provided an empty array or a non-array type object. The expected input is an array of Geometries.');
+    }
```

Comment:
We usually do not add such input validation checks. The reasoning behind this is to avoid verbose code and performance related issues.

I understand checks like that can improve the user experience to a certain degree since error messages become more informative. However, if we start here, we have to do the same for a lot of other methods and I think this gets quickly out of hand.

Reply:
I understand yea, it's up to you. It was suggested to me in the linked issue to create a PR for this so I did.

I know I for one would have loved having a nicer error message in this situation, but I do see why you would want to be consequent and keep it the same for all methods. I suppose it depends on how picky you make it, if little by little these things could improve with time when "someone complains", or if you want them all done at once or not at all :)

Either way, just glad to be here and thanks for reviewing 😄